samples/azure_files/readme.md

```diff
@@ -0,0 +1,11 @@
+# Using Azure Files
```

Comment:
If i remember correctly, azure files wont work on local machines (ie, VMs where your local box is not in azure) - is that still true? And if it is, we should make note that this sample will only work if your local R session is from an Azure VM (like a DSVM).

Reply:
Added some verbosity to the readme.